src/OrchardCore.Modules/OrchardCore.Users/Migrations.cs

```diff
@@ -17,7 +17,15 @@ public int Create()
                 .Column<int>("Count")
             );
 
-            return 1;
+            return UpdateFrom1();
```

Comment:
I like this, never thought about it, now I feel dumb for all the times I updated `Create` methods in the past.